book/website/ethical-research/activism/activism-case-study-google.md

```diff
@@ -0,0 +1,53 @@
+(er-activism-case-study-google)=
+# Case study: Google Workers & Project Maven
+
+Tech workers have historically had low levels of unionisation for a variety of different reasons. 
+Some have also been ambivalent about the potential impact of the companies and products they work for on systemic inequality and injustice across the globe.
```

Comment:
Is the point in this sentence that some tech workers don't mind that they work on problematic products? Or is it more linked to the next para, that some *do* mind and they are starting to speak up? 
There's a report from doteveryone on this in the UK, which it might be good to link to - what do you think? Miller C and Coldicutt R, ‘People, Power and Technology: The Tech Workers’ View’ (doteveryone 2019) <https://doteveryone.org.uk/report/workersview/>

Reply:
Have linked this down the bottom!